xenoglossia/builtins.py

```diff
@@ -214,6 +214,18 @@ def buttify(input, *args):
     """
     return re.sub("(?i)(butt|but)", u"\U0001F351", input)
 
+_DOTMATRIX_CHARS = ["⡮⢵", "⣟⣳", "⢎⣉", "⣏⡱", "⣟⣋", "⡯⠍", "⢎⣥", "⡗⢺", "⡇", "⢉⠏", "⡧⢎", "⣇⣀", "⡗⢺"
```

Comment:
No worries! The `u` prefix with the unicode characters inside the literal would have worked too, btw.